apps/tx-builder/src/lib/analytics.ts

```diff
@@ -0,0 +1,13 @@
+const SAFE_APPS_ANALYTICS_CATEGORY = 'safe-apps-analytics'
+
+export const trackSafeAppEvent = (action: string, label?: string) => {
+  window.parent.postMessage(
+    {
+      category: SAFE_APPS_ANALYTICS_CATEGORY,
+      action,
+      label,
+      safeAppName: 'Transaction Builder',
+    },
+    '*',
+  )
+}
```

Comment:
IMO this function needs to be exported to a library and then imported into apps so the app doesn't have to define how and what it sends

Reply:
yep agree, but let's see if we are going to use it more because right now we have it in the Wallet Connect app but other apps went out of scope right now from the current analytics requirements. I merged this because it was already done and could be useful